### PR TITLE
added center_of_mass and volume

### PR DESCRIPTION
**Pull request by obucklin**
this adds center of mass and volume to the `Assembly` class. These properties are useful for planning assembly with cranes or helicopters. 

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_timber.datastructures.Beam`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_timber/assembly/assembly.py [hunk 1]
```diff
@@ -201,3 +201,37 @@ def set_topologies(self, topologies):
...
+        return Point(*total_position) * (1.0 / total_vol)
+
+    @property
+    def volume(self):
+        """Returns the center of mass of the assembly.
+
+        Returns
+        -------
+        list
+            The center of mass of the assembly.
```
Comment (chenkasirer) [inline src/compas_timber/assembly/assembly.py:234]: copy paste accident?

File: src/compas_timber/assembly/assembly.py [hunk 2]
```diff
@@ -201,3 +201,37 @@ def set_topologies(self, topologies):
...
     def topologies(self):
         return self._topologies
+
+    @property
+    def center_of_mass(self):
+        """Returns the center of mass of the assembly.
+
+        Returns
+        -------
+        list
```
Comment (chenkasirer) [inline src/compas_timber/assembly/assembly.py:211]: should be a `Point`, no?

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+* Added `center_of_mass` property to Assembly class.
```
Comment (chenkasirer) [inline CHANGELOG.md:11]: missing space above this line

**Review by chenkasirer — CHANGES_REQUESTED** [review]

File: src/compas_timber/assembly/assembly.py (hunk 2 above)
Comment (obucklin) [reply src/compas_timber/assembly/assembly.py]: yeah I should have checked what copilot wrote....

fixed it now

File: CHANGELOG.md (hunk 3 above)
Comment (obucklin) [reply CHANGELOG.md:12]: done

**Review by chenkasirer — APPROVED** [review]
LGTM

**Pull request merged**